src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -166,35 +210,35 @@ const AssetContainer: React.FC<Props> = ({
 };
 
 const Wrapper = styled.div`
-  height: 558px;
   width: 100%;
 `;
 
-const AssetWrapper = styled(Flex)`
-  height: 425px;
+const AssetWrapper = styled.div<{ isSettingPage?: boolean }>`
+  height: ${({ isSettingPage }) => (isSettingPage ? "" : "425px")};
+  min-height: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const ButtonWrapper = styled.div<{ onRemove?: (assets: AssetType[]) => void }>`
+  display: flex;
+  justify-content: ${({ onRemove }) => (onRemove ? "flex-end" : "center")};
 `;
 
 const AssetList = styled(Flex)<{ layoutType?: LayoutTypes }>`
-  ${({ layoutType }) => layoutType === "list" && "flex-direction: column;"}
-  max-height: 458px;
+  ${({ layoutType }) => layoutType === "list" && "flex-direction: column;"};
+  // max-height: 458px;
```

Comment:
```suggestion
```

Don't leave any lines of code that are simply commented out. Since you're using git, you can always revert it.